src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -237,7 +224,8 @@ public async Task RunMode_DesignTimeOnlyDocument()
                 loader: TextLoader.From(TextAndVersion.Create(SourceText.From("class C2 {}"), VersionStamp.Create(), "design-time-only.cs")),
                 filePath: "design-time-only.cs",
                 isGenerated: false,
-                documentServiceProvider: new DesignTimeOnlyDocumentServiceProvider());
+                designTimeOnly: true,
+                documentServiceProvider: null);
```

Comment:
Do we need to pass the 'null' explicitly? That's the default, right?

Reply:
This is calling `DocumentInfo.Create`, there aren't any optional parameters.